Makefile

```diff
@@ -1,9 +1,16 @@
 SOURCE_FILES := $(shell test -e src/ && find src -type f)
+VERSION := $(shell sed --posix -n 's,^version = \"\(.*\)\",\1,p' Cargo.toml)
+
 
 policy.wasm: $(SOURCE_FILES) Cargo.*
 	cargo build --target=wasm32-wasi --release
 	cp target/wasm32-wasi/release/*.wasm policy.wasm
 
+artifacthub-pkg.yml: metadata.yml Cargo.toml
+	kwctl scaffold artifacthub \
+	  --metadata-path metadata.yml --version $(VERSION) \
+	  --questions-path questions-ui.yml --output  artifacthub-pkg.yml
+
```

Comment:
Please, add this [warning](https://github.com/kubewarden/rust-policy-template/blob/01173695123c7bbb358f11287c328f996f621317/Makefile#L9-L12) message in your Makefile.